src/main/java/org/infobourg/betterrss/models/RssFlux.kt

```diff
@@ -0,0 +1,19 @@
+package org.infobourg.betterrss.models
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+import java.time.LocalDateTime
+
+//@Document
+@Document(collection = "RssFlux")
+data class RssFlux(
```

Comment:
It's *RSS feed*. Need to change *Flux* to *Feed* in other places (repo, service).
```suggestion
@Document(collection = "RssFeed")
data class RssFeed(
```